### DISCUSSION TITLE
support custom directory for SQL File Queries for better dbt compatibility

### DISCUSSION THREAD

**Discussion by wearpants** [evidence-dev/evidence#2382] [General]
## Feature Description

It would be nice to be able to specify a custom directory to look for [SQL File Queries](https://docs.evidence.dev/core-concepts/queries/#sql-file-queries) instead of the hardcoded `sources`

## Goal of Feature

I'd like to point Evidence to my *compiled* dbt `analyses` directory in `jaffle/target/compiled/jaffle/analyses/`

Could be done in frontmatter or via an environment variable I guess?

## Current Solution

Copy & paste. Gross.

## Alternative Ideas

Could make `sources/` a symlink to the compiled analyses directory, but this is fragile on Windows.

## Are you interested in contributing this feature?

So busy. No JS. But maybe?

**Comment by mcrascal** [comment]
Thanks @wearpants, this is a good idea.
